scripts/app.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable indent */
+
+const collection = JSON.parse(localStorage.getItem('myBookList'));
```

Comment:
- [ ] I can't able to add new books and shows this error (below is the screenshot). Probably this is coming because you are fetching localstorage which is `null` in my case. Please fix that.

![Screenshot from 2021-07-06 14-24-41](https://user-images.githubusercontent.com/22434660/124573197-f39b3e00-de66-11eb-937d-86bfc3b3f7af.png)


if you are not getting any error. First try `localstorage.clear()` in console and then try, you'll get this error.